xdmod-appkernels.spec.in

```diff
@@ -51,6 +51,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/xdmod/
 %{_datadir}/xdmod/
 %{_docdir}/%{name}-%{version}__PRERELEASE__/
+%{_docdir}/%{name}-%{version}/
+%{_sysconfdir}/modules.d/appkernels.json
```

Comment:
This looks wrong to me. Shouldn't it be %{_sysconfdir}/xdmod/modules.d/appkernels.json ?